src/sections/DiscoveryParagraph/DiscoveryParagraph.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FunctionComponent } from 'react';
+import { Container, Row, Col } from 'react-grid-system';
+
+import Elephant from '~/assets/img/elephant.jpg';
+
+import './DiscoveryParagraph.scss';
+
+const DiscoveryParagraph: FunctionComponent = () => {
+    const className = 'discovery-paragraph';
+    return (
+        <div className={className}>
+            <Container>
+                <Row>
+                    <Col xs={12} md={12} lg={12}>
+                        <img
+                            src={Elephant}
+                            className={`${className}__image`}
+                        ></img>
+                    </Col>
+                    <Col xs={12} md={12} lg={8}>
+                        <p className={`${className}__paragraph`}>
+                            Lorem ipsum dolor sit amet consectetur adipiscing
```

Comment:
Vistiek cia komponentas, ta galetum jam duomenis is parento per propsa paduot ;)